src/state/AppState.ts

```diff
@@ -9,6 +9,8 @@ export interface AppState {
 }
 
 export interface FinishedGameState {
+    id?: string;
```

Comment:
nullable id?

Reply:
When saving is not finished. Will be updated after save is finished.

Reply:
When saving is not finished. Will be updated after save is finished.